docs/resourcebookings/resourcebookings.rst

```diff
@@ -0,0 +1,143 @@
+﻿Reserveringen van Middelen
+=============================
+
+Dit reserveringssysteem is ontworpen om het beheer en de planning van middelen te vereenvoudigen door een flexibele,
+geautomatiseerde en gebruiksvriendelijke ervaring te bieden. Of het nu via de backend of een partnerportaal gebeurt,
+gebruikers kunnen eenvoudig de beschikbaarheid bekijken, reserveringen maken en planningen beheren met minimale inspanning.
+
+**Belangrijkste functionaliteiten:**
+
+• Definieer het type reservering met een kalender die beschikbaarheid toont
+• Reserveer meerdere middelen tegelijk (alleen als ze allemaal beschikbaar zijn)
+• Maak voorlopige reserveringen zodat anderen een tijdslot kunnen kiezen
+• Partners kunnen reserveren via hun portaal
+• Tokenized URL’s geven toegang tot reserveringen voor partners zonder gebruikersaccount
+• Backend-gebruikers kunnen reserveringen rechtstreeks vanuit de backend beheren
+• Automatische reserveringscyclus met berekende statussen
+• Automatisch aanmaken en verwijderen van vergaderingen
+• Ingebouwde conflictdetectie bij overlappende reserveringen
+• Stel deadlines in om verdere wijzigingen te blokkeren
+
+**1) Gebruikersinstellingen**
+
+**1.1 Om een backendgebruiker toe te staan middelen te reserveren:**
+
+• Navigeer naar Instellingen > Gebruikers & Bedrijven > Gebruikers.
+• Selecteer een bestaande gebruiker of klik op Aanmaken om een nieuwe toe te voegen.
+• Ga naar het tabblad Toegangsrechten.
+• Wijs de rol toe: Resource Booking > User.
+
+.. image:: media/rb001.png
+
+**1.2 Om een backendgebruiker toe te staan boekingstypen en combinaties te configureren en achterstallige boekingen te wijzigen:**
+
+1. Navigeer naar Instellingen > Gebruikers & Bedrijven > Gebruikers.
+2. Selecteer een bestaande gebruiker of klik op Aanmaken om een nieuwe toe te voegen.
+3. Ga naar het tabblad Toegangsrechten.
+4. Wijs de rol toe: Resource Booking > Manage.
+
+.. image:: media/rb002.png
+
+**2) Vergadertypen configureren**
```

Comment:
Vergadertypen > Reserveringstypen, please look for anywhere else and replace it as well. likewise for the singular form (vergadertype)